README.md

```diff
@@ -6,87 +6,66 @@ Deis (pronounced DAY-iss) is an open source PaaS that makes it easy to deploy an
 applications on your own servers. Deis builds on [Kubernetes](http://kubernetes.io/) to provide
 a lightweight, [Heroku-inspired](http://heroku.com) workflow.
```

Comment:
Should we remove the link to heroku.com here? It's quite prominent. I don't think we want to encourage readers to visit there from the top of a README--it's too distracting.